.gitignore

```diff
@@ -29,8 +29,9 @@ _eric6project/Uranium.e4t
 .cache
 *.kate-swp
 
-## Eclipse+PyDev
+## Eclipse+PyDev incl. Mylyn
 .project
+.settings/org.eclipse.mylyn.tasks.ui.prefs
```

Comment:
And I don't know how other IDE's are storing their settings. It is possible that they also use .settings for their files.
To be sure it is only ignoring Mylyn I only added the new file here.